CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Removed
 - Removed `version` field in STAC Collections. Use [Version Extension](extensions/version/README.md) instead.
 - Removed `summaries` field from Catalogs. Use Collections instead.
+- Asset Types in Items. Use the asset's `role` instead.
```

Comment:
It seems like there is a slight chance that there could be confusion about removing Item asset object `type` i.e. media type. Is this more of a 'Fix' to the Item asset object description than a 'Remove'?

Reply:
I made this clearer now.